debian-jessie

```diff
@@ -1,5 +1,10 @@
 FROM debian:jessie
 
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y apt-utils apt-transport-https
+RUN apt-get remove -y resolvconf openresolv network-manager
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y apache2 build-essential curl dnsutils git gnupg iptables iptables-persistent libcurl4-openssl-dev libjpeg-dev libxml2-dev libxslt1-dev mysql-server ntpdate openssh-server openssl postfix postfix-mysql postfix-pcre procmail python-dev python-jinja2 python-lxml python-pgpdump python-pip python-virtualenv rsyslog spambayes sudo tor torsocks wget zlib1g-dev
+
 COPY . /own-mailbox/
 RUN cd /own-mailbox/; \
```

Comment:
On Wed, Jul 20, 2016 at 7:12 PM, Pierre Parent <notifications@github.com>
wrote:
>
> Not so sure , because it means any small modification of
> required-packages.sh will trigger re-download of all packages, which is
> very very boring especially when you have a slow connection.
>

I think that required-packages.sh does not change very often, does it?